supabase/migrations/20240629035932_fix_fields_db_convention.sql

```diff
@@ -0,0 +1,32 @@
+ALTER TABLE app_versions 
+RENAME COLUMN "minUpdateVersion" TO min_update_version;
+
+ALTER TABLE channels
+RENAME COLUMN "disableAutoUpdateUnderNative" TO disable_auto_update_under_native;
+
+ALTER TABLE channels
+RENAME COLUMN "enableAbTesting" TO enable_ab_testing;
+
+ALTER TABLE channels 
```

Comment:
missing changes in users tables, as say here https://github.com/Cap-go/capgo/issues/684

Reply:
it was agreed to postpone this users table change